### PR TITLE
chore: add test for exported types in npm package

### PR DESCRIPTION
**Pull request by kachick**
This PR add test for https://github.com/octokit/graphql-schema/pull/924

Extracted from the original PR because of adding dev dependency https://github.com/arethetypeswrong/arethetypeswrong.github.io/tree/add0a9e21aee1288a13826f3cd7682840acbdc25/packages/cli

----

### Before the change?

* 

### After the change?

* 

### Pull request checklist
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [x] No

----

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "lint": "prettier --check '{bin,lib,test}/**/*.{js,ts}' *.md package.json",
     "lint:fix": "prettier --write '{bin,lib,test}/**/*.{js,ts}' *.md package.json",
     "test": "node test",
-    "validate:ts": "tsc --noEmit --noImplicitAny test/typescript-validate.ts",
+    "validate:ts": "tsc --noEmit --noImplicitAny test/typescript-validate.ts && attw --pack . --ignore-rules cjs-resolves-to-esm",
```
Comment (kachick) [inline package.json:14]: Or put in `postvalidate:ts` maybe better?

**Comment by kachick** [comment]
How do you think about adding this tooling?

I think adding type assertions, as these tools help with the process of publishing a package. However, I can close this PR if there are any objections. 🙏

**Review by nickfloyd — APPROVED** [review]
Hey @kachick Thank you for this change and apologies for the massive delay on getting this approved.  This seems like a great addition to help us keep things "healthy" ❤️

**Comment by kachick** [comment]
Thank you! 🙏

**Pull request merged**